mlflow/sklearn/__init__.py

```diff
@@ -534,6 +534,10 @@ def predict_proba(self, *args, **kwargs):
         if hasattr(self.sklearn_model, "predict_proba"):
             return self.sklearn_model.predict_proba(*args, **kwargs)
 
+    def predict_log_proba(self, *args, **kwargs):
```

Comment:
It looks like there is one more predict variant: https://scikit-learn.org/stable/modules/generated/sklearn.naive_bayes.GaussianNB.html#sklearn.naive_bayes.GaussianNB.predict_joint_log_proba

Reply:
Oh great catch! This one has exactly same signature so adding that should be fine. In the future we may want to implement better delegation mechanism.